### PR TITLE
Jb49209

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last two commits belong to this PR

**Review by jusa — APPROVED** [review]

File: sdk-setup/src/sdk-manage [hunk 1]
```diff
@@ -1423,7 +1423,7 @@ synchronise_target() {
         maybe_times=
     fi
 
-    echo >&2 "Synchronising target to host"
+    echo >&2 "Synchronising target to host..."
```
Comment (vigejolla) [inline sdk-setup/src/sdk-manage:1426]: Have you considered adding --info=progress2 to rsync parameters?

Comment (martyone) [reply sdk-setup/src/sdk-manage:1426]: That would be nice but unfortunately it does not print nicely with output not directed into terminal.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**